src/util.ts

```diff
@@ -109,6 +109,9 @@ export function spawn(command: string, args?: Array<string> | null, options?: Sp
 }
 
 export async function getElectronVersion(packageData: any, packageJsonPath: string): Promise<string> {
+  if (packageData.config['electron-version']) {
+    return packageData.config['electron-version']
```

Comment:
In any case user will get clear error on electron-download.

Electron version is not documented at all, yeah... We should document it some day.